src/hashmap.c

```diff
@@ -183,6 +183,10 @@ static int hashmap_rehash(struct hashmap_base *hb, size_t table_size)
     hb->table_size = table_size;
     hb->table = new_table;
 
+    if (old_table == NULL) {
```

Comment:
Just for consistency with surrounding code:
```suggestion
    if (!old_table) {
```